<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WastageDsp</class>
 <widget class="QWidget" name="WastageDsp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>这里应该是个圆饼图</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>破损数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>总数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>破损比例</string>
   </property>
  </widget>
  <widget class="QLabel" name="wstNoLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>no</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>no</string>
   </property>
  </widget>
  <widget class="QLabel" name="wstRateLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>no</string>
   </property>
  </widget>
  <widget class="QLabel" name="tuliLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>图例</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
